google/cloud/aiplatform/training_jobs.py

```diff
@@ -1148,26 +1270,16 @@ def run(
             accelerator_type=accelerator_type,
         ).pool_specs
 
-        # create model payload
-        managed_model = None
-        if model_display_name:
-            utils.validate_display_name(model_display_name)
-
-            container_spec = gca_model.ModelContainerSpec(
-                image_uri=self._model_serving_container_image_uri,
-                predict_route=self._model_serving_container_predict_route,
-                health_route=self._model_serving_container_health_route,
-            )
-
-            managed_model = gca_model.Model(
-                display_name=model_display_name, container_spec=container_spec
-            )
-
         # make and copy package
         python_packager = _TrainingScriptPythonPackager(
             script_path=self._script_path, requirements=self._requirements
         )
 
+        managed_model = self._managed_model
+        if model_display_name:
```

Comment:
`managed_model` should be None in the case where `model_display_name` is not passed in to support not exporting a model from the training job.

Reply:
Can you create a bug to capture that as a unit test case?